hack/install-deps.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# Copyright 2014 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Dependencies:
+# runc:
+# - libseccomp-dev(Ubuntu,Debian)/libseccomp-devel(Fedora, CentOS, RHEL)
+# - libapparmor-dev(Ubuntu,Debian)/libapparmor-devel(Fedora, CentOS, RHEL)
+# containerd:
+# - btrfs-tools(Ubuntu,Debian)/btrfs-progs-devel(Fedora, CentOS, RHEL)
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"/..
+. ${ROOT}/hack/versions
+
+BUILD_IMAGE=golang:1.8
+RUNC_PKG=github.com/opencontainers/runc
+CNI_PKG=github.com/containernetworking/cni
+CNI_DIR=/opt/cni
+CNI_CONFIG_DIR=/etc/cni/net.d
+CONTAINERD_PKG=github.com/containerd/containerd
+
+# Install runc
+go get -d ${RUNC_PKG}/...
+cd ${GOPATH}/src/${RUNC_PKG}
+git checkout ${RUNC_VERSION}
+make
+sudo make install
+which runc
+
+# Install cni
+go get -d ${CNI_PKG}/...
+cd ${GOPATH}/src/${CNI_PKG}
+git checkout ${CNI_VERSION}
+./build
+sudo mkdir -p ${CNI_DIR}
+sudo cp -r ./bin ${CNI_DIR}
+sudo mkdir -p ${CNI_CONFIG_DIR}
+sudo bash -c 'cat >'${CNI_CONFIG_DIR}'/10-containerd-bridge.conf <<EOF
+{
+	"cniVersion": "0.2.0",
```

Comment:
Yeah, [cni example](https://github.com/containernetworking/cni#running-the-plugins) and [cri-o](https://github.com/kubernetes-incubator/cri-o/tree/master/contrib/cni) are also using this.